src/olympia/versions/models.py

```diff
@@ -543,6 +543,19 @@ def unreviewed_files(self):
             models.Q(version__addon__status__in=amo.UNDER_REVIEW_STATUSES,
                      status__in=under_review_or_lite))
 
+    def activity_log(self, action__in_filter=None):
+        from olympia.devhub.models import ActivityLog  # Becoz circular import.
+        alog = ActivityLog.objects.for_version(self)
+        if filter:
```

Comment:
that will teach me for using `filter` as a variable name - it didn't trigger a warning about an undefined name.  I've removed the `Version` methods here in the refactoring anyway.